README.md

```diff
@@ -1 +1,5 @@
-Nectar is common presentation layer functionalities for zbay project - it contains logic that connects frontend and backend
\ No newline at end of file
+Nectar is common presentation layer functionalities for zbay project - it contains logic that connects frontend and backend
+
+### Running integration tests
+
+DEBUG='waggle*, nectar*' jest communityJest.test.ts
```

Comment:
Shouldn't this command be more inclusive? I mean, communityJest.test.ts file name indicates we run tests for communities only